examples/karaf-dump-example/README.md

```diff
@@ -0,0 +1,49 @@
+# Apache Karaf Diagnostic Dump example
+
+## Abstract
+
+This example shows how to create a new dump provider service. This provider is automatically included in the dump created
+by the `dev:dump-create` command.
+
+The dump provider example actually takes a screenshot of client screen.
+
+## Build
+
+The build uses Apache Maven. Simply use:
+
+```
+mvn clean install
+```
+
+## Deployment
+
+On a running Karaf instance, install the dump bundle:
+
+```
+karaf@root()> bundle:install -s 
```

Comment:
Install bundle command should be `bundle:install mvn:org.apache.karaf.examples/karaf-dump-example/4.2.0-SNAPSHOT`?

Reply:
When starting the bundle there is an unresolved error 
```
karaf@root()> bundle:install -s mvn:org.apache.karaf.examples/karaf-dump-example/4.2.0-SNAPSHOT 
Bundle ID: 43
Error executing command: Error installing bundles:
	Unable to start bundle mvn:org.apache.karaf.examples/karaf-dump-example/4.2.0-SNAPSHOT: org.osgi.framework.BundleException: Unable to resolve org.apache.karaf.examples.karaf-dump-example [43](R 43.0): missing requirement [org.apache.karaf.examples.karaf-dump-example [43](R 43.0)] osgi.extender; (&(osgi.extender=osgi.component)(version>=1.3.0)(!(version>=2.0.0))) Unresolved requirements: [[org.apache.karaf.examples.karaf-dump-example [43](R 43.0)] osgi.extender; (&(osgi.extender=osgi.component)(version>=1.3.0)(!(version>=2.0.0)))]
```